package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "@amcharts/amcharts4": "^4.9.12",
     "@apollo/client": "^3.1.5",
-    "@cowprotocol/cow-sdk": "^0.0.14",
+    "@cowprotocol/cow-sdk": "file:.yalc/@cowprotocol/cow-sdk",
```

Comment:
once https://github.com/cowprotocol/cow-sdk/pull/67 is merged and a new version is published I will update this

Reply:
bumped